docs/use/setup-wallet/index.md

```diff
@@ -48,7 +48,7 @@ On Ethereum, the system supports [MetaMask](https://metamask.io/download.html),
 
 ![](./images/wallet-component-eth.png#width=40%;)
```

Comment:
Line 45:
![](./images/wallet-component-cf.png#width=40%;)
Change to:
![image](https://github.com/centrifuge/documentation/assets/90972248/6af6d88b-dd75-4985-86e6-a573120388b3)